### DISCUSSION TITLE
link prediction doubts

### DISCUSSION THREAD

**Discussion by ecastillot** [stellargraph/stellargraph#2036] [Q&A]
Hi,

I have a list of disconnected graphs and each graph have their own edge connections. I want to train a model with each graph to be able to predict  edge connections in a new graph. However, I don't want to predict links between disconnected graphs because this no have sense in my problem. Could you please guide me on this problem?

I thought to make a super graph with all disconnected graphs and follow the link prediction examples. However, I don't want to evaluate connections between disconnected graphs.

Thanks for your support

**Comment by stefanschutz** [comment]
Hi,

I have a similar use-case like yours. Did you come up with the idea to solve this? Thanks!
